docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md

```diff
@@ -0,0 +1,134 @@
+---
+id: microsoft-exchange-tracelogs
+title: Sumo Logic App for Microsoft Exchange Tracelogs
+sidebar_label: Microsoft Exchange Tracelogs
+description: This Sumo Logic App for Microsoft Exchange Tracelogs provides visibility into Delivered, Failed, Quarantined, Pending, and Spam messages.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/saas-cloud/istio.png')} alt="Thumbnail icon" width="50"/>
```

Comment:
wrong logo

Reply:
Thank you so much for this. Removed it as we don’t have the logo not on. C2C. Will update in both once I get it.